src/types.ts

```diff
@@ -399,3 +396,112 @@ export interface UpdatedBugTemplate {
 export const UpdatedBugTemplateSpec: ObjectSpec<UpdatedBugTemplate> = {
   bugs: array(object(UpdatedBugSpec)),
 };
+
+export interface Attachment {
+  data: base64;
```

Comment:
I think by the same token we should decode the base64 and convert it into a Buffer.

Reply:
Done